src/api/cart/addDiscountCodeAnonim.ts

```diff
@@ -0,0 +1,34 @@
+import { publicInstance } from '..';
+import { getAnonimToken } from '../authAnonim';
+
+type IAddDiscoutCode = {
+  cartId: string;
+  cartVersion: number;
+  discountCode: string;
+};
+
+export const addDiscountCodeAnonim = async ({
+  cartId,
+  cartVersion,
+  discountCode,
+}: IAddDiscoutCode) => {
+  const anonimToken = await getAnonimToken();
+  const { data } = await publicInstance.post(
+    `/me/carts/${cartId}`,
```

Comment:
Is it really so, that we are to send requests to the authorized user endpoint, while working with anonimous user?